### DISCUSSION TITLE
Keyboard shortcut to Show/Hide Overlays

### DISCUSSION THREAD

**Discussion by alejandroid80** [baffler/Transparent-Twitch-Chat-Overlay#67] [Ideas]
It could be a great feature, to be able to assign a keyboard shortcut to show or hide the overlays.

I'm loving this app, btw.

Thanks for your work!

**Comment by spokenmusic** [comment]
Hi! First of all thanks for this app, its great and it works like a charm, I'm helping a streamer that uses this when he's on his driving sim, the thing is that I was hoping there was a way to have a hotkey to turn on/off the visibility of this app, so he can read chat after he finishes with the race but don't have the chat always in the screen, if this can also be attached to a stream deck app would be great, if not that's ok, just a general hotkey will do the trick, thanks again!

**Comment by baffler** [comment]
I'll get this added in the next update
